Panels/Header.module.less

```diff
@@ -92,6 +93,13 @@
 		margin: @sand-header-slotbelow-margin;
 	}
 
+	&.shadowed {	
```

Comment:
Instead of importing `colors.less` directly, let's move this definition into `applySkins` mixin. :D That would be better.

Reply:
As I moved definition into applySkins, `Header.js` also fixed to apply `skinnable`.